test/unit/specs/components/staking/__snapshots__/PageProposal.spec.js.snap

```diff
@@ -0,0 +1,279 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PageProposal has the expected html structure 1`] = `
+<div
+  class="tm-page"
+  data-title="Proposal"
+>
+  <header
+    class="tm-page-header"
+  >
+    <div
+      class="tm-page-header-container"
+    >
+      <div
+        class="tm-page-header-text"
+      >
+        <!---->
+        <!---->
+        <div
+          class="tm-page-header-body"
+        >
+          <div
+            class="header-balance"
+          >
+            <div
+              class="top"
+            >
+              <div
+                class="icon-container"
+              >
+                <img
+                  class="icon"
+                  src="~assets/images/cosmos-logo.png"
+                />
+              </div>
+              <div
+                class="total-atoms top-section"
+              >
+                <h3>
+                  Total Steak
+                </h3>
+                <h2>
+                  0.0000…
+                </h2>
+              </div>
+              <div
+                class="unbonded-atoms top-section"
+              >
+                <h3>
+                  Available Steak
+                </h3>
+                <h2>
+                  0.0000…
```

Comment:
is this a description ? why does it have the `...` ?

Reply:
this is how we show numbers in the balance header!